### PR TITLE
pta: stats: fix compilation incompatible pointer warning 

### PR DESCRIPTION
**Pull request by clementfaure**

### CODE REVIEW

**Comment by etienne-lms** [comment]
I forgot to mention 
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>` for commit "pta: stats: fix compilation incompatible pointer warning".

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -118,6 +118,7 @@ jobs:
           _make CFG_REE_FS_TA_BUFFERED=y
           _make CFG_WITH_USER_TA=n
           _make CFG_{ATTESTATION,DEVICE_ENUM,RTC,SCMI,SECSTOR_TA_MGT}_PTA=y
+          _make CFG_WITH_STATS=y CFG_TA_STATS=y
```
[the remark this answers is not in the document]
Comment (jforissier) [reply .github/workflows/ci.yml:121]: I'm fine with that too.

[the remark this answers is not in the document]
Comment (clementfaure) [reply .github/workflows/ci.yml:121]: Do I have your Ack?

**Review by jforissier — COMMENTED** [review]
`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Review by etienne-lms — COMMENTED** [review]
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>` for commit "ci: compile stats PTA on QEMU/QEMUv8".

**Comment by clementfaure** [comment]
Tags applied + rebase

**Pull request merged**